### PR TITLE
Update xdai and wxdai token logo

### LINKED ISSUE #1179: The icon of DAI on xDAI chain is the same the icon of ETH
**Description**

The icon of DAI on xDAI chain is the same the icon of ETH

**How to Reproduce**
1. Go to cowswap.exchange
2. Connect a wallet while on xDai network
3. Pick xDai (ETH equivalent token)

**Expected behavior**
xDai should not have the ETH symbol

**Screenshots**

screenshot_2021-08-04_09-58-32

**Additional context**
Also, if it's relevant for the described byg, add information like the browser or OS.

### PR DESCRIPTION
**Pull request by alongoni**
# Summary

Fixes #1179 
Before:
image
After:
image

  # To Test

1. Open the page [`swap`](https://pr2122--gpswapui.review.gnosisdev.com/)
2. Connect a wallet while on xDai network
3. Pick xDai and wxDai
4. Check the token logos. Should look like the screenshot above.

### CODE REVIEW

File: src/custom/constants/tokens/index.ts [hunk 1]
```diff
@@ -24,7 +24,6 @@ export const ADDRESS_IMAGE_OVERRIDE = {
   [USDC_XDAI.address]: getTrustImage(USDC.address),
   // [USDT_XDAI.address]: getTrustImage(USDT.address),
   [WBTC_XDAI.address]: getTrustImage(WBTC.address),
-  [WXDAI.address]:
-    'https://raw.githubusercontent.com/1Hive/default-token-list/master/src/assets/xdai/0xe91d153e0b41518a2ce8dd3d7944fa863463a97d/logo.png',
+  [WXDAI.address]: 'assets/images/wxdai.png',
```
Comment (ramirotw) [inline src/custom/constants/tokens/index.ts:27]:
```js
import wxDaiLogo from 'assets/images/wxdai.png'

...
[WXDAI.address]: wxDaiLogo,
...
```

**Review by alfetopito — APPROVED** [review]
It look soooo good! I love it!

**Comment by elena-zh** [comment]
Looks great!
One tiny question: xDAI icon still looks like a broken one in the activity modal. Is this OK for this PR? 
image

**Review by elena-zh — APPROVED** [review]
I have forgotten that the issue with ETH/XDAI tokens in the activity modal is a known one #2006 and is not related to the current PR. 
The current changes are approved!

**Review by ramirotw — COMMENTED** [review]
😍

**Review by henrypalacios — APPROVED** [review]
I like it, as well as xDai's brand update.

**Pull request merged**